go.mod

```diff
@@ -1,78 +1,80 @@
 module github.com/cloudogu/k8s-host-change
 
-go 1.20
+go 1.22.0
```

Comment:
Toolchain defines the suggested version to be used for this module. I think it is automatically updated because of a dependency used here. However we can adjust the version here.